microsoft-edge/toc.yml

```diff
@@ -75,9 +75,11 @@
 # TOOLS =======================================================================
         - name: Tools
```

Comment:
An interesting TOC design idea that mirrors the UI and the [All Tools screenshot](https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/about-tools?branch=pr-en-us-1706)  is something like this:
Tools
... Inspect tool
... Device Emulation
... Elements tool
... Console tool
... Sources tool for JavaScript debugging
... More tools
... ... 3D View tool
... ... ...
... ... WebAuthn tool

Reply:
Implemented, in the sibling prototype PR.